build.gradle

```diff
@@ -66,11 +66,8 @@ task codeBuild {
     doLast {
         println "Task: Building triggers..."
         for (String path : ballerinaPackages) {
-            if (!(path.contains("asb") || path.contains("salesforce") || path.contains("asgardeo") || 
-            path.contains("hubspot") || path.contains("quickbooks"))) {
-                exec {
-                    commandLine 'sh', '-c', "${ballerinaDistributionPath}/bin/bal test ${path}"
-                }
+            if (!path.contains("asgardeo")) {
```

Comment:
Shall we set `specificTrigger` as asgardio in L49, so that we can omit all these if checks?

Reply:
Fixed with [e1a75cd74795b20d9438a2b53b3318af34b595d6](https://github.com/ballerina-platform/asyncapi-triggers/pull/153/commits/e1a75cd74795b20d9438a2b53b3318af34b595d6)